daemon/SecretsImpl/pluginwrapper.cpp

```diff
@@ -545,6 +564,9 @@ bool EncryptedStoragePluginWrapper::initialize(const QByteArray &masterLockKey)
                 QStringList snames;
                 result = secretNames(cname, &snames);
                 if (result.code() != Result::Succeeded) {
+                    qCWarning(lcSailfishSecretsDaemon) << "cannot get secret names for"
+                                                       << cname << ":"
+                                                       << result.errorMessage();
```

Comment:
Please also print the result code in the warning.

Reply:
I've added code wherever it apply. Thanks for the suggestion.